tools/UglyToad.PdfPig.ConsoleRunner/UglyToad.PdfPig.ConsoleRunner.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>7.3</LangVersion>
```

Comment:
I know you've probably already attempted something similar and I didn't understand fully. But is it not possible instead to bump this to latest and change the target framework on line 6 to something like .NET 6 instead?

Reply:
@EliotJones I think it would work too, I'm also confused as to why we have the issue to begin with tbh... 

What I did here is in order to get the same errors in VS, and fixing the error messages.

I think if we increase the dotnet version to 6 here we would also need to increase the dotnet version in the GitHub Action that runs the integration tests.

Really up to you ;)